app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -68,7 +70,7 @@ class AdministratorControlsActivityPresenter @Inject constructor(
         PROFILE_EDIT_FRAGMENT -> selectedProfileId.let { profileId ->
           if (extraControlsTitle != null) {
             activity.loadProfileEdit(profileId = profileId, profileName = extraControlsTitle)
-            if (isProfileDeletionDialogVisible && profileId != 0) {
+            if (isProfileDeletionDialogVisible) {
```

Comment:
The original condition `&& profileId != 0` should not be removed because it is meant to check if the profile is an admin. What you can instead to is the following:
```suggestion
            if (isProfileDeletionDialogVisible && profileId.internalId != 0) {
```